cloudevents/adapters/aws-sns.md

```diff
@@ -0,0 +1,54 @@
+# Amazon Simple Notification Service CloudEvents Adapter
+
+This document describes how to convert
+[AWS SQS messages][sns-messages]
+into CloudEvents.
+
+Amazon SNS MAY send a subscription confirmation, notification, or unsubscribe confirmation message to your HTTP/HTTPS endpoints.
+
+Each section below describes how to determine the CloudEvents attributes
+based on the specified type of SNS messages.
+
+### Subscription Confirmation
+
+| CloudEvents Attribute | Value                                           |
```

Comment:
Should there be an extension attribute for the subscription ARN?

Reply:
But we already put the subscription ARN into the `source` attribute if the message contains a subscription ARN . Do we need to add a duplicate one to the event?

Reply:
The source attribute has the topic ARN; I was under the impression that a subscription ARN was different. (Again, I don't know about these things...)